internal/wrappers/features/java/src/test/java/gradle/cucumber/Documentdefs.java

```diff
@@ -234,7 +234,7 @@ public void the_collection_must_be_empty() {
         Assert.assertEquals(0, k.getDocument().count_(world.index, world.collection, "{}"));
     }
 
-    @And("^I get a partial error$")
+    @And("^I should have a partial error$")
```

Comment:
Why did you change this? This step does not exist, it still "I get a partial error" in the feature file, maybe you forgot to change it as well?

Reply:
Mistake from my part